content/help/ops/perf/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Performance Tuning
+description: Istio performance, sizing and tuning guide.
+weight: 90
+force_inline_toc: true
+---
+## Istio Performance Tuning
+
+Istio's goal is to offer a performance-optimized configuration out of the box.
+However, each use case has different performance criteria. Thus, creating performance-optimized configurations for all use cases is difficult.
+
+The following table shows a few configuration options that affect performance:
+
+| Key | Default Value | Performance Impact | Recommendation |
+| --- | --- | --- | --- |
+| `global.proxy.concurrency` | `2` | An incorrect value can cause high sidecar resource use and latency. | Set this value to equal to the number of CPU cores allocated to the proxy. |
+| `pilot.keepaliveMaxServerConnectionAge` | `30m` | Uneven load distribution across Pilot replicas for a longer period of time. | Reduce this number if you want quicker balancing of load at the cost of increased connect/disconnect activity. |
+| `pilot.traceSampling` | `1.0` |  Latency and throughput impact due to tracing. | This can be reduced further from the default. If set to 0.0%, use `x-client-trace-id` header to trace specific requests. |
+| `global.disablePolicyChecks` | `true` | Tail latency impact due to policy. | Enable policy checks *only* if you are using [rate limiting](/docs/tasks/policy-enforcement/rate-limiting/) or another istio-policy feature.|
+| `mixer.telemetry.enabled` | `true` | High CPU usage and additional latency by telemetry | Telemetry is a core part of Istio. However if you are only using the networking API and have other means of getting metrics, `istio-telemetry` can be turned off for a significant reduction in CPU usage.|
+
+Another important configuration option that should be is the considered for tuning is the [`Sidecar resource`](/docs/reference/config/networking/v1alpha3/sidecar), which is disabled by default. In a large mesh, there will be a high memory usage by sidecars and high CPU by Pilot, if the sidecar scope includes the whole mesh. By enabling namespace isolation there should be a significant reduction in sidecar memory usage and pilot CPU usage.
+
+### Istio default profile vs. functionality
+
+In order to get a good out of the box performance, Istio 1.1 disabled several features by default.
+
+| Feature | Default | Description |
+| --- | --- | --- |
+| `global.proxy.accessLogFile` | `""` |  Proxy access logging is disabled because it contributes to high CPU and I/O. Set this to `/dev/stdout` to enable access logging.|
+| `mixer.adapters.stdio.enabled` | `false` |  Mixer access logging is disabled because it is a debug feature and it contributes to high CPU and I/O. Set this to `true` to enable access logging.|
+| `global.disablePolicyChecks` | `true` | Enable policy checks *only* if you are using rate limiting or another istio-policy feature.|
```

Comment:
this flag is mentioned earlier, do we need to duplicate it again here?

Reply:
It is good to re-enforce that this feature is disabled by default, but agree that description should be different than the previous table.